src/Vegetation/canopy_parameterizations.jl

```diff
@@ -240,7 +240,7 @@ dark respiration (`Rd`), and the moisture stress factor (`β`).
 See Table 11.5 of G. Bonan's textbook, Climate Change and Terrestrial Ecosystem Modeling (2019).
 """
 function net_photosynthesis(Ac::FT, Aj::FT, Rd::FT, β::FT) where {FT}
-    An = max(0, min(Ac * β, Aj) - Rd)
+    An = max(0, min(Ac, Aj) * β - Rd)
```

Comment:
This is correct: Rd already has the factor, while Ac and Aj do not. From Bonan's book:
"Other models impose biochemical limitations and indirectly reduce stomatal conduct- ance by reducing An as soil moisture stress increases (multiplying Vcmax and Jmax by betaw)."